<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>712</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Projects on mentor</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tasks on worker</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Workers on project</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Workers failed deadline</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="open_file">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Open file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="workerFIO_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Worker FIO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="projectName_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Project Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="task_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Task</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mentorFIO_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Mentor FIO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dateGave_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Date gave</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="deadline_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Deadline</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dateComplete_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Date Complited</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_to_file">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Add to file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineNumber_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Number Of Line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="edit_file">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Edit file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_from_file">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Delete from file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="show_file">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Show file</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>118</number>
     </attribute>
     <column>
      <property name="text">
       <string>Worker FIO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Project name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Task</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mentor FIO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date gave</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Deadline</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Completed</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
